wallet/src/main/scala/org/constellation/wallet/Utils.scala

```diff
@@ -158,7 +180,7 @@ object SignHelp {
     bytes2hex(signData(hash.getBytes())(privateKey))
 
   def signedObservationEdge(oe: ObservationEdge)(implicit kp: KeyPair): SignedObservationEdge =
-    SignedObservationEdge(hashSignBatchZeroTyped(Hashable.hash(oe), kp))
+    SignedObservationEdge(hashSignBatchZeroTyped(oe.hash, kp))
```

Comment:
Why? I thought we should remove all implicit usages of kryo instead of adding new ones.

Reply:
Forgot 🤦‍♂